recipes/vtk/all/conanfile.py

```diff
@@ -0,0 +1,176 @@
+import os
+import re
+import shutil
+
+from fnmatch import fnmatch
+from conans import ConanFile, CMake, tools
+
+class VTKConan(ConanFile):
+    name = "vtk"
+    version = "9.0.1"
```

Comment:
No version needed, as it is already set by the `config.yml` file.
```suggestion
```

Reply:
Done